README.md

```diff
@@ -53,3 +54,11 @@ Feel free to raise and fix issues.
 For any questions, join #oshc-dev on Slack. Get an invite [here](https://opensourcehelp.herokuapp.com/).
 
 Note : All design related tasks have reward associated with them.
+
+
+<!-- develop -->
+
+[travis-image]: https://img.shields.io/travis/dbader/node-datadog-metrics/master.svg?style=flat-square
+[travis-url]: https://travis-ci.org/OpenSourceHelpCommunity/OpenSourceHelpCommunity.github.io.svg?branch=master
```

Comment:
When I changed the link from master to develop 
![screenshot from 2017-07-22 11-50-08](https://user-images.githubusercontent.com/10944610/28488987-f7e8ce4c-6ed4-11e7-80d9-c7478744d3ba.png)
Is it meant to be or something else.